Ginger/GingerCoreNET/Drivers/CoreDrivers/Mobile/Appium/GenericAppiumDriver.cs

```diff
@@ -2716,8 +2716,8 @@ public override Point GetPointOnAppWindow(Point clickedPoint, double SrcWidth, d
                     }
                     else
                     {
-                        ratio_X = (SrcWidth / 2) / ActWidth;
-                        ratio_Y = (SrcHeight / 2) / ActHeight;
+                        ratio_X = SrcWidth / ActWidth;
```

Comment:
Need to see if we need to add also flag for this for doing the changes only for needed devices after testing with iPad and others